schema-bridge.graphql

```diff
@@ -55,6 +66,30 @@ type Token @entity {
   lastPriceBlockNumber: BigInt
 }
 
+# CrosschainToken is the representation of a token from a different chain.
+# For example, if you bridge USDC from Ethereum to Polygon, the token on Polygon is a CrosschainToken.
+# CrosschainToken might or might not be canonical. It refers to the counterpart of the token on the other chain.
+# If the token is wrapped on the current chain, CrosschainToken will be canonical on the opposite chain; and viceversa.
+type CrosschainToken @entity {
+  # We prepend the chainID because it is possible for a token to have the same address on multiple chains.
+  # ChainID is the id of the destination chain (not the one the subgraph is running on), and tokenID is the 
+  # address of the token on the current chain.
+  " {chainID}-{tokenID} "
```

Comment:
why use a seperate identifier for chains when we have the `network` enum already?

Reply:
I was thinking because of simplicity (bridge contracts already use this chainID to identify the different chains), and to avoid having to map our `network` enum to chainIDs.

It is not a new ID I’m inventing, but these: https://chainlist.org/